File: app/src/main/java/com/skeleton/mvp/ui/onboarding/landing/LandingView.java
package com.skeleton.mvp.ui.onboarding.landing;

import com.skeleton.mvp.ui.base.BaseView;


/**
 * Developer: Geetanjali Gupta
 */
public interface LandingView extends BaseView {

    /**
     * Navigate to login screen.
     */
    void navigateToSignInScreen();

    /**
     * Navigate to sign up screen.
     */
    void navigateToSignUpScreen();

}

File: app/src/main/java/com/skeleton/mvp/ui/onboarding/landing/LandingView.kt
package com.skeleton.mvp.ui.onboarding.landing;

import com.skeleton.mvp.ui.base.BaseView;


/**
 * Developer: Geetanjali Gupta
 */
interface LandingView : BaseView {

    /**
     * Navigate to login screen.
     */
    fun navigateToSignInScreen()

    /**
     * Navigate to sign up screen.
     */
    fun navigateToSignUpScreen()

    /**
     * Navigate to home screen.
     */
    fun navigateToHomeScreen()

}
